src/misc/msccl/msccl_lifecycle.cc

```diff
@@ -77,9 +80,52 @@ static const char* mscclSchedulerPathEnv = "MSCCL_SCHEDULER";
 static const char* mscclSchedulerDefaultPath = "libmsccl-scheduler.so";
 static const char* mscclAlgoDirEnv = "MSCCL_ALGO_DIR";
 static const char* mscclAlgoDefaultDir = "msccl-algorithms";
+static const char* azureVMDetectionAgent = "http://169.254.169.254/metadata/instance?api-version=2019-06-04";
```

Comment:
Change to mscclAzureVMDetectionAgent to keep aligned?

Reply:
agree. Already regulated.